Shape_detection/test/Shape_detection/CMakeLists.txt

```diff
@@ -45,23 +45,13 @@ if(TARGET CGAL::Eigen3_support)
     test_region_growing_on_point_set_3_with_sorting
     test_region_growing_on_polygon_mesh_with_sorting
     test_region_growing_on_degenerated_mesh)
-    target_link_libraries(${target} PUBLIC CGAL::Eigen3_support)
+    target_link_libraries(${target} PRIVATE CGAL::Eigen3_support)
   endforeach()
 
   set(RANSAC_PROTO_DIR CACHE PATH "")
-  if(NOT RANSAC_PROTO_DIR STREQUAL "")
-    add_definitions(-DPOINTSWITHINDEX -DCGAL_TEST_RANSAC_PROTOTYPE)
-    include_directories(${RANSAC_PROTO_DIR})
-    include_directories(${RANSAC_PROTO_DIR}/MiscLib/)
-    file(GLOB proto_src "${RANSAC_PROTO_DIR}/*.cpp")
-    file(GLOB proto_misc_src "${RANSAC_PROTO_DIR}/MiscLib/*.cpp")
-    add_library(libproto STATIC  ${proto_src} ${proto_misc_src})
-    add_executable(test_validity_sampled_data "test_validity_sampled_data.cpp")
-    target_link_libraries(test_validity_sampled_data libproto CGAL::CGAL CGAL::Data CGAL::Eigen3_support)
-  else()
-    add_executable(test_validity_sampled_data "test_validity_sampled_data.cpp")
-    target_link_libraries(test_validity_sampled_data CGAL::CGAL CGAL::Data CGAL::Eigen3_support)
-  endif()
+
+  add_executable(test_validity_sampled_data "test_validity_sampled_data.cpp")
+  target_link_libraries(test_validity_sampled_data CGAL::CGAL CGAL::Data CGAL::Eigen3_support)
```

Comment:
The support for `RANSAC_PROTO_DIR` is removed. Is it intentional?

Reply:
yes. After a discussion with @soesau

Reply:
Right. That seems to be commit 9edf6eadc7a93ef451b184d0b2cc3355fcec9412. Thanks.